plugin/collector/snap-plugin-collector-mock2/mock/mock.go

```diff
@@ -56,14 +58,24 @@ func (f *Mock) CollectMetrics(mts []plugin.MetricType) ([]plugin.MetricType, err
 	rand.Seed(time.Now().UTC().UnixNano())
 	metrics := []plugin.MetricType{}
 	for i := range mts {
-		if c, ok := mts[i].Config().Table()["long_print"]; ok && c.(ctypes.ConfigValueBool).Value {
+		if c, ok := mts[i].Config().Table()["long_stdout_log"]; ok && c.(ctypes.ConfigValueBool).Value {
 			letterBytes := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
 			longLine := []byte{}
-			for i := 0; i < 8193; i++ {
+			for i := 0; i < bufio.MaxScanTokenSize; i++ {
 				longLine = append(longLine, letterBytes[rand.Intn(len(letterBytes))])
 			}
-			fmt.Println(string(longLine))
+			fmt.Fprintln(os.Stdout, string(longLine))
 		}
+
+		if c, ok := mts[i].Config().Table()["long_stderr_log"]; ok && c.(ctypes.ConfigValueBool).Value {
+			letterBytes := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+			longLine := []byte{}
+			for i := 0; i < bufio.MaxScanTokenSize; i++ {
```

Comment:
Is the goal here to append to longline until it is larger than the MaxScanTokenSize?